sapl-maven-plugin/src/test/java/io/sapl/mavenplugin/test/coverage/stubs/SaplMavenPluginTestMavenProjectStub.java

```diff
@@ -39,7 +39,7 @@ public SaplMavenPluginTestMavenProjectStub() {
         MavenXpp3Reader pomReader = new MavenXpp3Reader();
         Model           model;
         try {
-            model = pomReader.read(new XmlStreamReader(new File(getBasedir(), "pom.xml")));
+            model = pomReader.read((new FileReader(new File(getBasedir(), "pom.xml"))));
```

Comment:
See Sonar. And why the change ?

Reply:
I upgraded commons.io to version 2.15 and saw that the way XmlStreamReader is instantiated here is deprecated. I thought that doing without the class and thus without the import would be fine/even better.